src/modes/simple_select.js

```diff
@@ -82,13 +91,15 @@ module.exports = function(ctx, options = {}) {
         return ctx.store.get(id) !== undefined;
       }));
 
-      // Any mouseup should stop box selecting and dragging
-      this.on('mouseup', () => true, function() {
+      // Attach a listener to mouseout, pending the mapbox-gl-js release including
+      // https://github.com/mapbox/mapbox-gl-js/pull/2777
+      mapCanvas.addEventListener('mouseout', handleMouseout);
```

Comment:
This is a good idea, lets use `this.on('mouseout', () => true, function() {});` rather than a canvas map handler.